profile/profile.yaml

```diff
@@ -0,0 +1,135 @@
+# Your skills can be anything software related (e.g. language, frameworks, methodologies, software, etc.).
+skills:
+  # At least 5 skills
+  proficient:
+    - OOP
+    - Java
+    - Maven
+    - Linux/Bash
+    - Git/GitHub
+    - RDBMS/SQL
+    - Agile/Scrum
+  # At least 5 skills
+  competent:
+    - Eiffel
+    - Docker
+    - SpringBoot
+    - REST APIs
+    - C
+    - Go
+    - HTML
+    - CSS
+    - JavaScript
+    - Android Development
+  # At least 5 skills
+  familiar:
+    - Pandas/Jupyter/Data Analysis
+    - Blockchain
+    - Operating Systems
+    - AI/ML
+    - CyberSecurity
+
+# List of Jarvis projects
+jarvis_projects:
+  - name: "Cluster Monitor"
+    git_url: "/linux_sql"
+    # The project description allows hiring managers to understand your project and technologies you used(e.g. programing langauge, libraries, algorithms, hardwards, tools, etc.).
+    # The project description must start with an action verb (https://bit.ly/3guO98u)
+    description: "Developed a monitoring agent to keep track of a cluster's resources and usage across time; this provided the team invaluable information about decisions to scale the cluster (and each node in it) up or down. The project was implemented with Bash, PostgreSQL, SQL, and Docker."
+  - name: "Core Java Apps"
+    git_url: "/core_java"
+    description:
+      - "Grep App: Searches for a text pattern recursively in a given directory and outputs matched lines to a file - implemented with Java 8 Streams."
+      - "JDBC App: Performs fundamental CRUD operations using the DAO pattern as an abstraction layer with DTOs as the actual objects encapsulating data to and from the data source."
+      - "Twitter App: Posts/shows/deletes Twitter posts via the Twitter REST API - designed with the well-known MVC (minus V) architecture."
+  - name: "SpringBoot App"
+    git_url: "/springboot"
+    description: "Developed a RESTful API that simulates an online stock trading system; it retrieves stock market data from IEX Cloud and persists it into a PSQL instance; this data can then be used to CRUD quote, trader, and order data against the database. This API can be consumed by front-end and mobile developers, as well as traders. The application is implemented as a microservice using SpringBoot (to manage dependencies and provide the web-server), PSQL (to persist and CRUD data using DAO), and IEX Cloud (as the stock market data source); it also falls under the MVC and 3-tier architecture - it has a client tier (HTTP clients to consume API), an application tier (SpringBoot Java program which processes data), and a data tier (where data is persisted)."
+  - name: "Data Wrangling"
+    git_url: "/python_data_wrangling"
+    description: "Implemented a data analysis project to help a retail organization drive their revenue back up; used a PostgreSQL instance as a data warehouse and Python Pandas/Jupyter to perform OLAP operations on the data and answer necessary business questions."
```

Comment:
yes, you want to help the company to improve its revenue. Specifically, you want to help the company to segment customers in order to design tarted marketing strategies.